src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+			    showJudgementTypeList();
+			}
+		});
+		
+		submitNoSamplesPane.add(selectJudementTypesButton);
+		
+		JLabel lblNewLabel_2 = getWhatsThisOne();
+		submitNoSamplesPane.add(lblNewLabel_2);
+		
+		JPanel submitNoSamplesPane_1 = new JPanel();
+		FlowLayout flowLayout_2 = (FlowLayout) submitNoSamplesPane_1.getLayout();
+		flowLayout_2.setAlignment(FlowLayout.LEFT);
+		submitNoSamplesPane_1.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+		submitNoSamplesPane_1.setBounds(33, 161, 650, 32);
+		controlsPane.add(submitNoSamplesPane_1);
+		   
+        checkBoxProblems = new JCheckBox("Problems");
+        checkBoxProblems.setVerticalAlignment(SwingConstants.TOP);
+        checkBoxProblems.setToolTipText("Select Problems");
+        checkBoxProblems.setSelected(true);
+        submitNoSamplesPane_1.add(checkBoxProblems);
+        
+        submitAllProblemsRadioButton = new JRadioButton("Submit All");
+        submitAllProblemsRadioButton.setSelected(true);
+        submitNoSamplesPane_1.add(submitAllProblemsRadioButton);
+        
+        submitSelectedProblemsRadioButton = new JRadioButton("Submit Selected");
+        submitNoSamplesPane_1.add(submitSelectedProblemsRadioButton);
+        
+        selectedProblemsLabel = new JLabel("None selected");
+        selectedProblemsLabel.setToolTipText("None selected");
+        submitNoSamplesPane_1.add(selectedProblemsLabel);
+        
+        selectProblemsButton = new JButton("Select");
+        selectProblemsButton.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent e) {
+                showProblemList();
+            }
+        });
+        selectProblemsButton.setToolTipText("Select Problems");
+        submitNoSamplesPane_1.add(selectProblemsButton);
+        
+        JPanel submitLanguages = new JPanel();
+        FlowLayout fl_submitLanguages = (FlowLayout) submitLanguages.getLayout();
+        fl_submitLanguages.setAlignment(FlowLayout.LEFT);
+        submitLanguages.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+        submitLanguages.setBounds(33, 191, 650, 32);
+        controlsPane.add(submitLanguages);
+        
+        checkBoxLanguage = new JCheckBox("Languages");
+        checkBoxLanguage.setVerticalAlignment(SwingConstants.TOP);
+        checkBoxLanguage.setToolTipText("Select Languages");
+        checkBoxLanguage.setSelected(true);
+        submitLanguages.add(checkBoxLanguage);
+        
+        submitAllLanguagesRadioButton = new JRadioButton("Submit All Languages");
+        submitAllLanguagesRadioButton.setSelected(true);
+        submitLanguages.add(submitAllLanguagesRadioButton);
+        
+        submitSelectedLanguagesRadioButton = new JRadioButton("Submit Selected");
+        submitLanguages.add(submitSelectedLanguagesRadioButton);
+        
+        selectedLanguagesLabel = new JLabel("None selected Lang?");
```

Comment:
As a user, the GUI presented by this label is incomprehensible to me.  What I see is:

![image](https://github.com/pc2ccs/pc2v9/assets/9256691/c45c5562-96dd-4b17-ae8b-b3ba3b56b0b7)

I have NO idea what `None selected Lang?` is trying to convey.

If it's trying to convey that the radio button indicates I'm specifying that (only) the selected languages are to be submitted then (1) the `Lang?` portion of the label should be removed, and (2) it would be a lot clearer if the `None selected` portion was in parentheses (indicating it is information subordinate to, but related to, the `Submit Selected` button.  Like:
`<radiobutton> Submit Selected (None selected) <SelectButton>`

Reply:
I consider this an approval showstopper.